src/devtools/client/debugger/packages/devtools-reps/src/object-inspector/actions.js

```diff
@@ -166,6 +159,7 @@ async function releaseActors(state, client, dispatch) {
 
 function invokeGetter(node, receiverId) {
   return async ({ dispatch, client, getState }) => {
+    return
```

Comment:
Is this supposed to be a bailout?

Reply:
yeah - i don't want to think about that now :)